### PR TITLE
Rephrased the string to shorten its length in strings.xml

### LINKED ISSUE #3146: Reword "Show the date and time when the file was modified last time"
"Show the date and time when the file was modified last time" to "Show the date and time of last modification" (less words) or "Show the date and time when the items was modified last time", reported as issue in Transifex.

### PR DESCRIPTION
**Pull request by ashway171**
#### Description
Rephrased the line "Show the data and time when the file was modified last time " in strings.xml to "Show the data and time of last modification" thus reducing words.

#### Issue tracker   

Fixes #3146

#### Automatic tests

- [ ] Added test cases
  
#### Manual tests
- [ ] Done  
  

#### Build tasks success  

Successfully running following tasks on local:
- [ ] `./gradlew assembledebug`
- [ ] `./gradlew spotlessCheck`

### CODE REVIEW

**Review by VishnuSanal — CHANGES_REQUESTED** [review]
Please delete the old `String` & create a new `String` so that `Transifex` can pick that up.

**Review by EmmanuelMess — APPROVED** [review]

**Comment by EmmanuelMess** [comment]
> Please delete the old `String` & create a new `String` so that `Transifex` can pick that up.

Meaning was not changed, translations should be mostly the same AFAIK.

**Comment by ashway171** [comment]
So, should I go for the changes?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -153,7 +153,7 @@
     <string name="thumb_summary">Show Thumbnails of apps and images</string>
     <string name="thumb">Show Thumbnails</string>
     <string name="hidden">Show Hidden Files and Folders</string>
-    <string name="lastModifiedSummary">Show the date and time when the file was modified last time</string>
+    <string name="lastModifiedSummary">Show the data and time of last modification</string>
```
Comment (VishnuSanal) [inline app/src/main/res/values/strings.xml:156]: This is `date` itself!

**Review by VishnuSanal — CHANGES_REQUESTED** [review]

**Comment by VishnuSanal** [comment]
> So, should I go for the changes?

Nope

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (ashway171) [reply app/src/main/res/values/strings.xml]: So sorry, fixing it right away

**Review by VishnuSanal — APPROVED** [review]

**Pull request merged**